.github/workflows/pr_test_when_merged.yml

```diff
@@ -37,11 +37,8 @@ jobs:
           registry-url: 'https://registry.npmjs.org'
       - name: Install dependencies
         run: yarn
-      - name: Test Success
-        run: yarn test
```

Comment:
once this line is removed, we don't run the test in the release pipeline?

Reply:
In the `publish` job, you can find the needs.
```
needs: run_test_on_mainnet
```

This ensures that the `run_test_on_mainnet` job must complete successfully before this job will run.
So i removed useless duplicated test.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds

Reply:
(discussed offline)
thanks for the explanation!